suncityla/README.md

```diff
@@ -34,3 +34,40 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
 The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
 
 Check out our [Next.js deployment documentation](https://nextjs.org/docs/app/building-your-application/deploying) for more details.
+
+## Prisma ORM
+
+This project uses [Prisma](https://prisma.io) as an ORM. You can find the Prisma schema in `prisma/schema.prisma`.
+
+To get started, start a local postgres server, create a database and call it SunCityLA.
+An example of the connection URL is in the `.example.file` file.
```

Comment:
Hi @fmtabbara,

First and foremost, great job, and thank you again for the video call earlier today! I really learned a lot from our discussion.

I managed to get everything set up, but it was a bit of a challenge due to missing instructions. Would it be possible for you to update the documentation with details on how to set up the database locally? It would be really helpful to mention pgAdmin and how to connect it to the local database. It took me quite a bit of time to figure it out, and I can provide some resources you can include.

Here’s a good starting point for the setup:

Everyone should install Docker to speed up the process.
Then, you can follow these commands (Answer #1 here: [link](https://stackoverflow.com/questions/25540711/docker-postgres-pgadmin-local-connection)).
Also, when introducing new environments or technologies in your PR, could you include a brief explanation of what they are and why they’re needed for development? That would make it easier for everyone to stay aligned.

Other than that, I ran all the commands and they all function well.